src/config/locales/en/psychological.js

```diff
@@ -38,7 +38,7 @@ export const psychological = {
   },
   competence: {
     heading: {
-      occurred: 'Provide the date this occurred',
+      occurred: 'Provide the date this occurred ok ok ',
```

Comment:
Assuming this wasn't meant to be included?

Reply:
Ha! Nope, thanks! That was some leftover testing :)